providers/base/bin/camera_test.py

```diff
@@ -653,22 +873,41 @@ def add_device_parameter(parser):
             help=("Use the /dev/videoN where N is the lowest value available"),
         )
 
+    # Detect subparser
     subparsers.add_parser("detect")
+
+    # Led subparser
     led_parser = subparsers.add_parser("led")
     add_device_parameter(led_parser)
-    display_parser = subparsers.add_parser("display")
-    add_device_parameter(display_parser)
-    still_parser = subparsers.add_parser("still")
-    add_device_parameter(still_parser)
-    still_parser.add_argument(
-        "-f", "--filename", help="Filename to store the picture"
+
+    # Video subparser
+    video_parser = subparsers.add_parser("video")
+    add_device_parameter(video_parser)
+    # add a quiet option false by default
+    video_parser.add_argument(
+        "-q",
+        "--quiet",
+        action="store_true",
+        help=("Don't display video, just run the camera"),
```

Comment:
So `quiet` is not quiet, it is `headless`? Can we rename this? We already have Checkbox calling automated "silent", please lets not create another weird alias

Reply:
super confusing as we also have a log level!

Reply:
Renamed quiet parameter to headless